config/sql/se/Ebooks.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE `Ebooks` (
+  `EbookId` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Identifier` varchar(511) NOT NULL,
+  `Created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `Updated` timestamp NOT NULL,
+  `WwwFilesystemPath` varchar(511) NOT NULL,
+  `RepoFilesystemPath` varchar(511) NOT NULL,
+  `KindleCoverUrl` varchar(511) NULL,
+  `EpubUrl` varchar(511) NULL,
+  `AdvancedEpubUrl` varchar(511) NULL,
+  `KepubUrl` varchar(511) NULL,
+  `Azw3Url` varchar(511) NULL,
+  `DistCoverUrl` varchar(511) NULL,
+  `Title` varchar(255) NOT NULL,
+  `FullTitle` varchar(255) NULL,
+  `AlternateTitle` varchar(255) NULL,
+  `Description` varchar(511) NOT NULL,
+  `LongDescription` text NOT NULL,
```

Comment:
Add a fulltext index to this and any other `text` columns so we can search them

Reply:
Done, but see the note above about the trouble I had using the fulltext index.